src/Makefile.test.include

```diff
@@ -60,6 +60,7 @@ BITCOIN_TESTS =\
   test/merkle_tests.cpp \
   test/miner_tests.cpp \
   test/multisig_tests.cpp \
+  test/net_tests.cpp \
```

Comment:
Ideally CAddrDB and other code which interacts with the datadir should have its own file, but currently its is integrated with net.cpp. 
I have some deserialization tests that belong in addrman_tests.cpp, but not in this pull request.